### DISCUSSION TITLE
Realtime example?

### DISCUSSION THREAD

**Discussion by chrisallenmoore** [spinspire/pocketbase-sveltekit-starter#4] [Ideas]
Hi, will you be giving an example of realtime capabilities with PocketBase and sveltekit?

**Comment by jkdoshi** [comment]
Yes, certainly. I'll be doing that. Actually, I already have it in another project. Just haven't added it to this project.

I'll post back here once it's added (shortly).

**Comment by jkdoshi** [comment]
@chrisallenmoore we now have an example of realtime capabilities with PocketBase and sveltekit. See [posts/+page.svelte](https://github.com/spinspire/pocketbase-sveltekit-starter/blob/master/frontend/src/routes/posts/%2Bpage.svelte). It uses a Svelte store to reactively reflect the contents of a collection. It even has pagination built-in, but I haven't yet added pagination to the UI.

**Comment by chrisallenmoore** [comment]
This sounds great! I’ll check it out!

Thanks so much!
